python_modules/dagster-graphql/dagster_graphql/schema/pipelines/pipeline.py

```diff
@@ -349,7 +349,7 @@ class GrapheneRun(graphene.ObjectType):
     stepKeysToExecute = graphene.List(graphene.NonNull(graphene.String))
     runConfigYaml = graphene.NonNull(graphene.String)
     runConfig = graphene.NonNull(GrapheneRunConfigData)
-    mode = graphene.NonNull(graphene.String)
+    mode = graphene.Field(graphene.String)
```

Comment:
if you look below we have `mode=DEFAULT_MODE_NAME` so im not sure how it could be null

Reply:
also have to deal with the        ` interfaces = (GraphenePipelineRun,)` bit think these fields conflict now.

Reply:
I suspect this weirdness has something to do with `GraphenePipelineRun`. Do we even need it? The only place it's used is in `GrapheneRun`. Perhaps we can just delete it and fold any fields it has that aren't also in `GrapheneRun` in?